packages/storybook8/stories/Components/ParticipantList/ParticipantList.story.tsx

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import { ParticipantList as ParticipantListComponent, ParticipantListParticipant } from '@azure/communication-react';
+import { Stack } from '@fluentui/react';
+import React from 'react';
+
+const ParticipantListStory: (args) => JSX.Element = (args) => {
+  const participantsControls = [...args.remoteParticipants, ...args.localParticipant];
+
+  const mockParticipants: ParticipantListParticipant[] = participantsControls.map((p, i) => {
+    return {
+      userId: `userId ${i}`,
+      displayName: p.name,
+      state: p.status,
+      isMuted: p.isMuted,
+      isScreenSharing: p.isScreenSharing,
+      isRemovable: true
+    };
+  });
+
+  const myUserId = mockParticipants[mockParticipants.length - 1].userId;
+
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  const onParticipantRemove = (_userId: string): void => {
+    // Do something when remove a participant from list
```

Comment:
Suggestion: In the image overlay we display an alert pop up, maybe we can do something like that so no need to force the eslint disable